spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/procedures/ExpireSnapshotsProcedure.java

```diff
@@ -101,14 +103,18 @@ public InternalRow[] call(InternalRow args) {
         action.retainLast(retainLastNum);
       }
 
-      if (maxConcurrentDeletes != null && maxConcurrentDeletes > 0) {
```

Comment:
maxConcurrentDeletes > 0 is already checked in the `preconditions` above.

Reply:
It's best to leave changes like this for a separate cleanup pr